### PR TITLE
EDU-3007: Add logging level information

### PR DESCRIPTION
**Pull request by fairlydurable**
Aligned among all SDKs except PHP, which isn't in alignment with observability

### CODE REVIEW

File: docs/develop/dotnet/observability.mdx [hunk 1]
```diff
@@ -117,9 +117,23 @@ Spans are created and serialized through the server to give one trace for a Work
 
 **How to log from a Workflow to Temporal .NET SDK**
 
-Send logs and errors to a logging service, so that when things go wrong, you can see what happened.
-
-The SDK core uses `WARN` for its default logging level.
+Logging enables you to record critical information during execution.
```
Comment (jsundai) [inline docs/develop/dotnet/observability.mdx:120]: Is execution referring to Workflow Execution?

File: docs/develop/typescript/observability.mdx [hunk 2]
```diff
@@ -222,7 +240,7 @@ Runtime.install({
 });
 ```
 
-A common use case for this is to write log mesages to a file to be picked up by a collector service, such as the [Datadog Agent](https://docs.datadoghq.com/logs/log_collection/nodejs/?tab=winston30).
+A common use case for this is to write log messages to a file to be picked up by a collector service, such as the [Datadog Agent](https://docs.datadoghq.com/logs/log_collection/nodejs/?tab=winston30).
```
Comment (jsundai) [inline docs/develop/typescript/observability.mdx:243]: nice catches!

File: docs/develop/dotnet/observability.mdx (hunk 1 above)
Comment (fairlydurable) [inline docs/develop/dotnet/observability.mdx:120]:
```suggestion
Logging enables you to record critical information during code execution.
```

File: docs/develop/python/observability.mdx [hunk 3]
```diff
@@ -87,6 +87,24 @@ Spans are created and serialized through the server to give one trace for a Work
 
 ## Log from a Workflow {#logging}
 
+Logging enables you to record critical information during execution.
```
Comment (fairlydurable) [inline docs/develop/python/observability.mdx:90]:
```suggestion
Logging enables you to record critical information during code execution.
```

File: docs/develop/typescript/observability.mdx [hunk 4]
```diff
@@ -106,11 +106,29 @@ Similarly, you can customize the OpenTelemetry `NodeSDK` propagators by followin
 
 ## Log from a Workflow {#logging}
 
+Logging enables you to record critical information during execution.
```
Comment (fairlydurable) [inline docs/develop/typescript/observability.mdx:109]:
```suggestion
Logging enables you to record critical information during code execution.
```

File: docs/develop/java/observability.mdx [hunk 5]
```diff
@@ -128,9 +128,23 @@ For more information, see the Temporal [OpenTracing module](https://github.com/t
 
 ## Log from a Workflow {#logging}
 
-Send logs and errors to a logging service, so that when things go wrong, you can see what happened.
-
-The SDK core uses `WARN` for its default logging level.
+Logging enables you to record critical information during execution.
```
Comment (fairlydurable) [inline docs/develop/java/observability.mdx:131]:
```suggestion
Logging enables you to record critical information during code execution.
```

File: docs/develop/go/observability.mdx [hunk 6]
```diff
@@ -197,12 +197,26 @@ OpenTracing and OpenTelemetry are natively supported by [Jaeger](https://www.jae
...
 ## Log from a Workflow {#logging}
-
 **How to log from a Workflow using the Go SDK.**
 
 Send logs and errors to a logging service, so that when things go wrong, you can see what happened.
 
-The SDK core uses `WARN` for its default logging level.
+Loggers create an audit trail and capture information about your Workflow's operation.
+An appropriate logging level depends on your specific needs.
+During development or troubleshooting, you might use debug or even trace.
```
Comment (jsundai) [inline docs/develop/go/observability.mdx:206]: On the preview, it seems like debug, trace, info, warn should have command ticks on it. It looks a little odd without it.

**Review by jsundai — APPROVED** [review]
Left a comment about command ticks but unblocking this.

**Comment by fairlydurable** [comment]
@jsundai Updating all levels to use code face (back ticks). Thank you

**Pull request merged**